woocommerce/woocommerce
# E2e/fix failing daily product variations

### All Submissions:






### Changes proposed in this Pull Request:

`merchant/create-variable-product.spec.js` has been failing on our daily runs.  It looks like there might have been some updates to the UI that changed the order of form fields.

Closes [#37069 ](https://github.com/woocommerce/woocommerce/issues/37069)



### How to test the changes in this Pull Request:

1. Pull branch
2. `pnpm run env:test`
3. `pnpm run test:e2e-pw`
4. Verify that run completes successfully.
5. Verify that e2e tests pass on CI (on this PR) as well.




### Other information:




### FOR PR REVIEWER ONLY:
